src/curated-corpus/components/ScheduleItemForm/ScheduleItemForm.tsx

```diff
@@ -227,7 +233,7 @@ export const ScheduleItemForm: React.FC<
           >
             <Grid item xs={12}>
               <Box mt={3}>
-                <Accordion>
+                <Accordion defaultExpanded={expandSummary}>
```

Comment:
This setting lets us expand the Accordion component if necessary without having to manage its state manually.